src/raftstore/store/debug.rs

```diff
@@ -186,6 +209,38 @@ impl Debugger {
         compact_range(db, handle, start, end, false);
         Ok(())
     }
+
+    /// Set a region to tombstone by manual.
+    pub fn tombstone_region(&self, region: u64, conf_ver: u64) -> Result<()> {
```

Comment:
Mark a region as tombstone doesn't (and shouldn't) require a running TiKV instance, otherwise the updated region will probably be overwritten by the running instance.

Reply:
Yes, so there is no `tombstone_region` API in `server/service/debug.rs`, and tombstone command in `tikv-ctl.rs` doesn't support remote mode either. I just put the code here. If it's confuse, which place to place this function is better?

Reply:
The function name confused me, which looks like a get function, but here is a set function, a bad name.

Reply:
Maybe we should get the current region epoch from PD and then set the tombstone with it.

Reply:
can you use another PR to support this feature?

This PR is only for refactor.

Reply:
Got it. I will raise this in next PR, pass a `PDClient` into this function, and move this code to `tikv-ctl.rs`.